### ISSUE TITLE
Error from server

### ISSUE THREAD

**Issue by EmilienSanterre** [KleeGroup/focus-components#1206]
# Current behaviour

No warnings when I have an error from the server on a field which is in the form
See https://github.com/KleeGroup/focus-components/blob/43ed4a403b525c5847dd923f9c3055db66a24d8f/src/common/mixin/store-change-behaviour.js#L77

# Expected behaviour

Add  awarning or a `toLowerCase` in the comparison

**Comment by GuenoleK** [comment]
@EmilienSanterre what about now ? Do you still need this enhancement ?

**Comment by EmilienSanterre** [comment]
Yes, something like "An error on the field "key" was returned but the field was not found in the form" would be useful.

**Comment by Hartorn** [comment]
Add console.warn if no ref found to set the error
